esmvaltool/diag_scripts/runoff_et/catchment_analysis.py

```diff
@@ -475,7 +475,8 @@ def get_catchment_data(cfg):
     """
     catchments = get_defaults()
     catchments['refname'] = 'default'
-    catchment_filepath = cfg.get('catchmentmask')
+    catchment_filepath = os.path.join(cfg['auxiliary_data_dir'],
+                                      cfg.get('catchmentmask'))
```

Comment:
add check that the aux dir actually exists in the config, if not raise:
```
if not cfg.get("auxiliary_data_dir", None):
    raise ValueError("Need to specify auxiliary_data_dir in config-user.yml")
if not cfg.get('catchmentmask', None):
    raise ValueError("Need catchmen mask in recipe")
```
then check for the existence of file:
```
if not os.path.isfile(catchment_filepath):
    raise IOError("Not a file %s" % catchment_filepath)
```

Reply:
Looks good to me, thanks for implementing. Maybe the error message for the missing catchment mask could point to the recipe documentation where its download link is stored.

Reply:
Thank you for your suggestions!

@valeriupredoi the check on `auxiliary_data_dir` is not required, since it is set to a default path in `esmvalcore/_config.py` if not specified.